### PR TITLE
New PCB - Curiosity - Alice Layout PCB

### PR DESCRIPTION
**Pull request by sidcarter**
## Description

New PCB - Curiosity - Alice Layout PCB

* Uses STM32 ARM MCU
* Supports MX switches
* Has LED Indicators
* Has underglow
* Does not have in-switch LEDs
* Includes Default keymaps, and two personal keymaps

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by chapter63** [comment]
Guessing I can incorporate this change when this gets merged - https://github.com/qmk/qmk_firmware/pull/7705

**Comment by zvecr** [comment]
I plan to do at least a 2nd pass of all the boards so it could always wait till then. However theres nothing stopping you from updating the rules.mk right now if you want to rebase/merge master.

**Comment by sidcarter** [comment]
done. I tried that last night and didn't work, so didn't add that in. But obvo, a PR was merged for that mcu selection and code builds now. yay!

so here we are - thanks @zvecr 👍

File: keyboards/cheshire/curiosity/keymaps/crd/keymap.c [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```
Comment (fauxpark) [inline keyboards/cheshire/curiosity/keymaps/crd/keymap.c]:
```suggestion
```

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**